tests/frontend/test_ops_decompositions.py

```diff
@@ -798,8 +798,10 @@ def test_Zgate_decomposition(self, hbar, tol):
         Z = ops.Zgate(p)
         cmds = Z.decompose(prog.register)
         assert isinstance(cmds[0].op, ops.Dgate)
-        assert np.allclose(cmds[0].op.p[0], alpha, atol=tol, rtol=0)
-        assert np.allclose(cmds[0].op.p[1], 0, atol=tol, rtol=0)
+
+        r = cmds[0].op.p[0]
```

Comment:
Just for safety reasons
```suggestion
       assert len(cmds) == 1
        r = cmds[0].op.p[0]
```